### ISSUE TITLE
The rest of the book

### ISSUE THREAD

**Issue by slabko** [rougier/python-opengl#66]
I didn't mean to be intrusive, but I was just wondering if you plan to continue working on the book to finish the rest of the chapters. I really enjoy it, and I would definitely love to see the continuation.

In any case, love you work!

**Comment by rougier** [comment]
Thanks, and many thanks for your sponsoring. I've tried to contact you but did not find your email :) I plan to finish at some point because I'm working on vispy 2.0 and the book will be adapted accordingly. By the way, you might also be interested in 
https://matplotlib.org/matplotblog/posts/custom-3d-engine/

**Comment by slabko** [comment]
Good point! I guess nobody wanted to write to me before :) Time to update my profile. Just in case, the email is andrew@slabko.com.

I read the Custom 3D engine in Matplotlib some time ago. That was the second thing I read from you - after the Python to NumPy book. Loved it!

I asked because I think you have a good perspective on the use of framebuffers. Considering what you wrote in the Numpy book, I thought there might be some examples where each frame depends on the state of the previous one. 

No pressure :) I thought you had some good ideas in that area, and since framebuffer introduces some interesting possibilities, I'm pretty sure your work will be another eye-opener. I am looking forward to the continuation!

**Comment by rougier** [comment]
Yes, you can do amazing thing with the framebuffer. For the numpy examples, it's based on an old tutorial from gpgpu (general programming on the GPU) and uses the "ping pong" techniques. In the glumpy examples directory, you've also somepost-processing filters made with the framebuffer. I think there are also some example with render to texture.

**Comment by nvictor** [comment]
I am also very interested in the rest of this book. I also noticed that your focus has switched to your other visualization book. But I want to be one of the voices supporting and looking forward to seeing more of this book.

**Comment by rougier** [comment]
Thanks. I'm actually more or less back to to 3D and I hope to finish it but no schedule at this point.
